docs/radio.rst

```diff
@@ -0,0 +1,67 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string.
+
+.. py:function:: receive()
+
+    Receive the next incoming message on the message queue. Returns ``None`` if
+    there are no pending messages.
+
+.. py:function:: config(length=32, queue=3, channel=7, power=0, address=b'uBit\x00', data_rate=radio.MODE_1MHZ)
+
+    Configures the various settings relating to the radio. The specified
+    default values are sensible.
+
+    The ``length`` defines the size, in bytes, of a message sent via the radio.
+    It can be up to 256 (????? CHECK THIS from memory) bytes long.
+
+    The ``queue`` specifies the number of messages that can be stored on the
+    incoming message queue.
+
+    The ``channel`` can be an integer value between 0 and 100 (inclusive) that
+    defines an arbitrary "channel" to which the radio is tuned. Messages will
+    be sent via this channel and only messages received via this channel will
+    be put onto the incoming message queue. Each step is 1MHz wide, based at
+    2400MHz.
+
+    The ``power`` is an integer value between 0 and 7 to indicate the strength
+    of signal used when broadcasting a message. The higher the value the
+    stronger the signal, but the more power is consumed by the device.
+
+    The ``address`` is an arbitrary name, expressed as bytes, that's used to
+    filter incoming packets at the hardware level, keeping only those that
+    match the address you set. The default used by other micro:bit related
+    platforms is ``b'uBit\x00'``.
```

Comment:
It looks like the base and prefix are combined here, maybe there should be more info about the restrictions around this value. At the very least indicate that it should be 40 bits / 5 bytes.